config/routes.rb

```diff
@@ -57,11 +59,30 @@
     end
   end
 
+  # CMS  -----------------------------------------------------------------------
+  constraints !Rails.application.cms? do # NB: enabled if false
```

Comment:
Is this backwards?

Reply:
Can you explain how this is supposed to work -- cms is only enabled when it is set to false?

Reply:
No, I was going to ask you. It went in originally without being inverted and didn't work, then on a whim I banged it and it did, hence the comment. It's going to be something silly. Any ideas?

Reply:
Nope, I'm clueless here - I don't think I ever tested the opposite, so sounds like I had it accidentally working with the ENV var. Let me have a play with it, but for now I think we need to get this into main....